lib/parser.ypp

```diff
@@ -1787,7 +1789,7 @@ DLLLOCAL void yyerror(YYLTYPE* loc, yyscan_t scanner, const char* str) {
 %destructor { qore_class_private::get(*$$)->deref(); } class_attributes
 %destructor { free($$); } IDENTIFIER ANGLE_IDENTIFIER VAR_REF SELF_REF CONTEXT_REF COMPLEX_CONTEXT_REF BACKQUOTE SCOPED_REF SCOPED_VREF KW_IDENTIFIER_OPENPAREN QORE_CAST CLASS_STRING optname ident_openparen
```

Comment:
`HASHDECL_IDENTIFIER_OPENCURLY` is missing here.

Reply:
nice catch - I will push the fix after the other pull requests have been merged to avoid too many jenkins cycles, so we only have one more build for this branch before merging (hopefully)